yarn.lock

```diff
@@ -2160,7 +2161,7 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@automattic/ui@workspace:packages/ui":
+"@automattic/ui@workspace:^, @automattic/ui@workspace:packages/ui":
```

Comment:
I'm kinda curious why this change happens. Are we not consistently referencing this dependency?